packages/signatures/build.js

```diff
@@ -0,0 +1,50 @@
+import { writeFile } from 'node:fs/promises';
+import { resolve } from 'node:path';
+import { parseAbiItem, toFunctionSelector } from 'viem';
+import events from './manifests/events.json' with { type: 'json' };
+import functions from './manifests/functions.json' with { type: 'json' };
+
+const index = {
+  events: {
+    abi: {},
+    selectors: {},
+  },
+  functions: {
+    abi: {},
+    selectors: {},
+  },
+};
+
+function addToIndex(type, signature, target) {
+  if (signature.startsWith('//') || signature.startsWith('*')) return;
```

Comment:
This would need to be inside of the actual manifest entry not necessarily at start of line, right? Example might be useful

Reply:
sure, idea is we could chunk manifest for specific integrations
```json
[
  "// Zora Mint",
  "mint(...)",
  "// ...more comments"
]
```